ansible/ext-inventory.yml

```diff
@@ -0,0 +1,65 @@
+# This playbook updates data at ooni-sysadmin.git/ext/
+---
+- hosts: localhost
+  connection: local
+  gather_facts: false
+  tasks:
+    - name: get git-root
+      command: git rev-parse --show-toplevel
+      register: extroot
+      changed_when: false
+
+    - set_fact:
+        extroot: '{{ extroot.stdout }}/ext'
+        socks5: '{{ 32767 | random(1024) }}'
+        jumphost: '{{ namecheap_jumphosts | random }}'
+    - name: ssh jumphost
+      command: ssh -D {{ socks5 }} {{ jumphost }} sleep 60
```

Comment:
I avoid `uri` module and use curl with a proxy for a reason -- I don't want a secret token that may be used to update DNS records to be kept on the disk of jumphost as a temporary file.
I'll add a comment about that.